backend/orchestration.go

```diff
@@ -194,8 +191,8 @@ func (w *orchestratorProcessor) applyWorkItem(ctx context.Context, wi *Orchestra
 		}
 	}
 
-	if added == 0 {
-		w.logger.Warnf("%v: all new events were dropped", wi.InstanceID)
+	if len(wi.State.NewEvents) == 0 {
+		w.logger.Warnf("%v: no new events to process", wi.InstanceID)
```

Comment:
Should this be a warning? Not a Debug?

Reply:
I think a warning its fine, because this case should be exceptional, and should almost never happen, and if it happens is due to retries or other underlying errors